src/stemmer.h

```diff
@@ -6,9 +6,31 @@
 extern "C" {
 #endif
 
+typedef enum LANGUAGE {
```

Comment:
no need for enum label here

Reply:
It fails without. You might mix C and C++